setup.py

```diff
@@ -671,6 +671,41 @@ def add_multiarch_paths(self):
         finally:
             os.unlink(tmpfile)
 
+    def add_wrcc_search_dirs(self):
+        # add library search path by wr-cc, the compiler wrapper
+
+        def convert_mixed_path(path):
+            # convert path like C:\folder1\folder2/folder3/folder4
+            # to msys style /c/folder1/folder2/folder3/folder4
+            drive = path[0].lower()
+            left = path[2:].replace("\\", "/")
+            return "/" + drive + left
+
+        cc = sysconfig.get_config_var('CC')
+        tmpfile = os.path.join(self.build_temp, 'wrccpaths')
+        if not os.path.exists(self.build_temp):
+            os.makedirs(self.build_temp)
```

Comment:
If I recall correctly, makedirs() can be called even if the path already exists:

```suggestion
        os.makedirs(self.build_temp)
```

Reply:
Yes. But needs extra argument exist_ok to indicate. Otherwise it will report exception if the path already exists as the default value of exist_ok is False.

Reply:
Oh right, I forgot that. Maybe keep the exists() test, it's ok.